apps/admin-gui/src/app/users/pages/user-detail-page/user-accounts/user-accounts.component.ts

```diff
@@ -39,7 +40,7 @@ export class UserAccountsComponent implements OnInit {
         (vos) => {
           this.vos = vos;
           if (this.vos.length) {
-            this.loadMember(this.vos[0]);
+            this.loadMember(this.vos.sort(compareFnName)[0]);
```

Comment:
Could you please also fix the subscribe methods (three occurrences) to a not deprecated version?